src/store/store.ts

```diff
@@ -1232,6 +1237,9 @@ export const useStore = defineStore('store', {
     setAmalgamatingBusinessesValid (valid: boolean) {
       this.stateModel.amalgamation.amalgamatingBusinessesValid = valid
     },
+    setAmalgamationCourtApproval (courtApproval: boolean) {
+      this.stateModel.amalgamation.courtApproval = courtApproval
+    },
```

Comment:
I thought we had a ticket to implement the court approval component, but I can't find it. Can you? If not, please create one in the epic that we can groom tomorrow.

Reply:
Isn't it this one Sev? https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/18749

Reply:
Yes, thanks. (I've updated its title to make it easier to find next time 😉 )

I guess you can check off a couple of items in that ticket.

Reply:
Sounds good, will do that.